frontend/src/audio.js

```diff
@@ -0,0 +1,32 @@
+/*!
+ * sri sri guru gaurangau jayatah
+ */
+
+import Alpine from 'alpinejs';
+import './player';
+
+document.addEventListener('alpine:initialized', () => {
+  let article = document.querySelector('#audio-page');
+  /** type @{HTMLElement} */ // JSDoc types
+  if (article instanceof HTMLElement) {
```

Comment:
Negate the condition and exit early instead of a long if.

Reply:
negated if condition and exited..